src/test/java/core/basesyntax/service/operation/impl/BalanceOperationHandlerTest.java

```diff
@@ -0,0 +1,45 @@
+package core.basesyntax.service.operation.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.dto.FruitTransactionDto;
+import core.basesyntax.service.FruitService;
+import core.basesyntax.service.impl.FruitServiceImpl;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class BalanceOperationHandlerTest {
+    private Storage storage;
+    private FruitService fruitService;
+    private BalanceOperationHandler handler;
+
+    @BeforeEach
+    void setUp() {
+        storage = new Storage();
+        fruitService = new FruitServiceImpl(storage);

```

Comment:
I do not think we need service here, it is unused

Reply:
not resolved

Reply:
Yes , but I can't create new BalanceOperationHandler(fruitService) without fruitService.